### PR TITLE
fix: rename some repositories owner

### PR DESCRIPTION
**Pull request by suzuki-shunsuke**
They were transferred from https://github.com/suzuki-shunsuke to https://github.com/aquaproj

https://docs.github.com/en/repositories/creating-and-managing-repositories/transferring-a-repository

## Check

```console
$ git grep suzuki-shunsuke
.github/workflows/ts.yaml:      # https://github.com/suzuki-shunsuke/github-comment
README.md:- name: suzuki-shunsuke/github-comment@v4.0.0 # renovate: depName=suzuki-shunsuke/github-comment
aqua.yaml:  - name: suzuki-shunsuke/github-comment@v4.0.1
```

### CODE REVIEW

File: .github/renovate.json5 [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "extends": [
     "config:base",
-    "github>suzuki-shunsuke/aqua-renovate-config",
+    "github>aquaproj/aqua-renovate-config",
```
Comment (suzuki-shunsuke) [inline .github/renovate.json5:4]: https://github.com/aquaproj/aqua-renovate-config

File: README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # aqua-action [![ts](https://github.com/int128/aqua-action/actions/workflows/ts.yaml/badge.svg)](https://github.com/int128/aqua-action/actions/workflows/ts.yaml)
 
-This is an action to install packages using https://github.com/suzuki-shunsuke/aqua.
+This is an action to install packages using https://github.com/aquaproj/aqua.
```
Comment (suzuki-shunsuke) [inline README.md:3]: https://github.com/aquaproj/aqua

File: README.md [hunk 3]
```diff
@@ -10,7 +10,7 @@ Create `aqua.yaml` and add packages.
 ```yaml
 registries:
 - type: standard
-  ref: v0.10.10 # renovate: depName=suzuki-shunsuke/aqua-registry
+  ref: v0.10.10 # renovate: depName=aquaproj/aqua-registry
```
Comment (suzuki-shunsuke) [inline README.md:13]: https://github.com/aquaproj/aqua-registry

**Review by int128 — APPROVED** [review]
Thanks.

**Pull request merged**